content/pt-br/service-discovery.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Descoberta de Serviço
+status: Concluído
+category: conceito
+tags: ["networking", "", ""]
+---
+
+## O que é
+
+A descoberta de serviço é o processo de localização de instâncias individuais que compõem um serviço.
+Uma ferramenta de descoberta de serviço rastreia os vários nós ou terminais que compõem um serviço.
```

Comment:
Sounds better to me "... rastreia os vários nós ou *endpoints* que compõem ...". Think about it.